### ISSUE TITLE
source and dest endpoint identical

### ISSUE THREAD

**Issue by relihanl** [digidotcom/python-xbee#6]
in the explicit message object passed to the explicit data received callback, the "source_endpoint" and "dest_endpoint" are the same. It appears that the source endpoint is being copied from the dest_endpoint

i.e. in the following both printed values are identical

```python
def my_expl_data_received_callback(explicit_xbee_message):
         LOGGER.info("Source endpoint:        %s"
                 % utils.hex_to_string(utils.int_to_length(explicit_xbee_message.source_endpoint)))
         LOGGER.info("Destination endpoint:   %s"
                 % utils.hex_to_string(utils.int_to_length(explicit_xbee_message.dest_endpoint)))

```

**Comment by relihanl** [comment]
Before anyone asks, I'm using a XB240ZB configured as a ZigBee Coordinator. F/W is 21A7

**Comment by rubenmoral** [comment]
Hi @relihanl,

Thanks for reporting this problem, we have just fixed it in `master` (f420d8711b1fc305e1d5da7e62638a822ac14a62).

Best regards.
